player-style/transformed-data/.~lock.aggdf.csv#

```diff
@@ -0,0 +1 @@
+,dima,dima-Inspiron-7786,03.08.2020 19:23,file:///home/dima/.config/libreoffice/4;
```

Comment:
Should this file be in the repo?

Reply:
ill need to remove it

Reply:
ok I removed that lock file which wasnt needed